webpack-dev.config.js

```diff
@@ -19,20 +20,27 @@ const config = {
       { test: /\.svg$/, use: [ { loader: 'svg-sprite-loader' } ] },
       {
         test: /\.scss$/,
-        use: [
-          'style-loader',
-          'css-loader?modules&importLoaders=1&localIdentName=cw__[name]_[local]',
-          'resolve-url-loader',
```

Comment:
did you omit `style-loader` && `resolve-url-loader` on purpose?

Reply:
hmm, I used the construct from "production" webpack file, as wanted to get the same main.css file as after `compile` command.

Reply:
I checked and we're not taking advantage of them, at least of the `resolve-url-loader` anywhere and the difference between `style-loader` and `css-loader` is a bit unclear for me. And if it wasn't in the production config then i'd say it's reduntant now

Reply:
Previous config was taken from styleguide webpack config, so I think it was just a standard one when using this library.